app/lib/features/pins/actions/edit_pin_actions.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:acter/common/widgets/edit_html_description_sheet.dart';
+import 'package:acter/common/widgets/edit_title_sheet.dart';
+import 'package:acter/features/pins/Utils/pins_utils.dart';
+import 'package:acter/features/pins/providers/pins_provider.dart';
+import 'package:acter_flutter_sdk/acter_flutter_sdk_ffi.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/l10n.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+
+void showEditPintTitleDialog(
+  BuildContext context,
+  WidgetRef ref,
+  ActerPin pin,
+) {
+  showEditTitleBottomSheet(
```

Comment:
How would these resolve to desktop ? Generally, no desktop UI contains bottomsheets but instead a modal window pop-up. And having bottomsheets for desktop are considered bad ux-approach!

Reply:
This how it looks in desktop.
<img width="952" alt="Screenshot 2024-08-16 at 8 43 16 PM" src="https://github.com/user-attachments/assets/ffa853dc-7e59-4cb7-90f5-4bb27252b306">

There are many other places where it is the same. Don't degree with you in UX approach for desktop. @gnunicorn also mentioned regarding the same earlier [here](https://github.com/acterglobal/a3/pull/2064#discussion_r1717108047).

Let's come to some agreement and would love to do changes accordingly on all the places with separate PR. :)

Reply:
I care only 1/10 about whether that get another UI on desktop. I care 9/10 that we agree on a solution and stick to it. So whatever you two can agree on, is fine with me.

Reply:
@gnunicorn 
Let's go with @gtalha07's suggestion as it make sense to me for better UI on desktop.

https://github.com/acterglobal/a3/issues/2074
Created above issue and will do require changes at all relevant places.